streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamThread.java

```diff
@@ -266,7 +268,7 @@ void retryWithBackoff(final Map<TaskId, Set<TopicPartition>> tasksToBeCreated, f
                     } catch (final LockException e) {
                         // ignore and retry
                         if (!retryingTasks.contains(taskId)) {
-                            log.warn("{} Could not create task {}. Will retry: ", logPrefix, taskId, e);
+                            log.warn("{} Could not create task {} due to {}. Will retry: ", logPrefix, taskId, e);
```

Comment:
Are you sure this work as intended? IIRC correctly, there was an PR to remove unused `{}` markup. From my understanding, `log.warn` will not consider parameter `e` to be replaced within the error message string because it's the last parameter an of type exception. Thus, the third `{}` will be printed as-is without any substitution. Let me know if my understanding is incorrect.